static/views/companies.html

```diff
@@ -152,7 +152,8 @@ <h3 style="margin-top: 10px;">2-Click Apply</h3>
               </article>
           {{ end }}
           <article class="line-item" id="email-subscribe-banner" style="margin-top:10px;">
-		  <div style="width: 50%;font-size:15pt;text-align:center;font-weight:bold;display:table;margin:20px auto;" id="email-subscribe-item-text">Get a weekly email with all new Go jobs</div>
+              <div style="text-align: center; font-weight: bold; color:#000090">Join {{ .EmailSubscribersCount }}+ others</div>
+		  <div style="width: 50%;font-size:15pt;text-align:center;font-weight:bold;display:table;margin:5px auto 20px auto;" id="email-subscribe-item-text">Get a weekly email with all new Go jobs</div>
                 <input type="email" name="job-email-subscription" id="job-email-subscription" style="width:50%;margin:10px auto;display:table;border: 1px solid #d9d9d9;" placeholder="Your Email">
                 <input type="submit" onclick="subscribe()" id="email-subscribe-item-btn" value="Subscribe" style="width:50%;display:table;margin:5px auto;"/>
```

Comment:
actually it would be enough and much simpler to replace everywhere if we added this here :)
```suggestion
                <input type="submit" onclick="subscribe()" id="email-subscribe-item-btn" value="Join {{ .EmailSubscribersCount }}+ others" style="width:50%;display:table;margin:5px auto;"/>
```

Can you give me write access to your repo branch https://github.com/dsantos/golang.cafe/tree/subscribers-count? I will push this for you :)

Reply:
Done

Reply:
Thank you!!